### ISSUE TITLE
Accessibility issue: <html> element must have a lang attribute on /az_core/mobile_nav_callback/menu_link_content%3A482538b7-50d5-4e89-b18c-7405f238fbf1

### ISSUE THREAD

**Issue by accesswatch** [accesswatch/qs-proposal#50]
## What
An accessibility scan flagged the element `<html>` because <html> element must have a lang attribute. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/html-has-lang?application=playwright.

[View screenshot](https://github.com/accesswatch/qs-proposal/blob/gh-cache/.screenshots/68966b46-45a3-4dcd-bafe-4986a833dbee.png)

To fix this, ensure every html document has a lang attribute.

Specifically:

Fix any of the following:
-   The <html> element does not have a lang attribute

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
